Backend/contents.js

```diff
@@ -0,0 +1,70 @@
+require("dotenv").config();
+const express = require("express");
+const app = express();
+const cors = require("cors");
+const accountsDb = require("./db/accounts");
+const bodyParser = require('body-parser');
+var urlencodedParser = bodyParser.urlencoded({ extended: false })
+app.use(cors());
+app.use(bodyParser.json());
+
+const accounts = [{
+        id: "1",
+        name: "argam",
+        creationDate: "05.12.21",
+        owner: "argam",
+        action: "view",
+    },
+    {
+        id: "2",
+        name: "argam",
+        creationDate: "12.11.20",
+        owner: "argam",
+        action: "view",
+    },
+    {
+        id: "3",
+        name: "argam",
+        creationDate: "12.12.20",
+        owner: "argam",
+        action: "view",
+    },
+];
+
+function giveSingleAccount(id) {
+    const account = accounts.find((oneAccount) => oneAccount.id == id);
+    return account;
+}
+
+
+app.get("/api/accounts", async (req, res) => {
+    if (process.env.IS_MOCK == true) {
```

Comment:
move mock logic to top level. And have separate files for Db Connect and mock data